package.json

```diff
@@ -40,5 +40,8 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.18.0",
     "eslint-plugin-react-hooks": "^1.7.0 || ^4.0.0"
+  },
+  "dependencies": {
+    "eslint-plugin-simple-import-sort": "^7.0.0"
```

Comment:
Should this not be a part of `devDependencies`?

Reply:
IMO, this belongs in dependencies. The reason being `dependencies` specifies packages that are needed to **use** or **run** the package, while `devDependencies` is for packages that are needed to **build** or **develop** the package. In production you wouldn't install `devDependencies`. 

Since this package itself is a development tool, it can get a bit murky, but since this is a plugin directly being used by this config, and this config won't work without that plugin, I feel it belongs in dependencies.

Reply:
Ah sweet, Thanks for the clarification.

Reply:
Please follow the pattern here - this is set up according to best practices for eslint. The consuming library is responsible for defining all of these as dependencies based on the peer dependencies defined here. They're in dev dependencies because they're really only used in dev to test the package. 

Once you add something here, it needs to be added as a dependency in frontend-build.

Reply:
Is there a reason this plugin is different than the others?